services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsExcludeTest.java

```diff
@@ -0,0 +1,133 @@
+package com.mapbox.api.directions.v5.models;
+
+import com.mapbox.api.directions.v5.DirectionsCriteria;
+import com.mapbox.geojson.Point;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+public class RouteOptionsExcludeTest {
+  @Test
+  public void parseTwoPointsWithTwoCriteriaMixed() {
```

Comment:
NIT Normally is good to structure the tests following the triple A 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40 http://wiki.c2.com/?ArrangeActAssert

```java
@Test
public void parseTwoPointsWithTwoCriteriaMixed() {
    // Arrange
    
    // Act
    
    // Assert
}
```

Note the empty lines between Arrange / Act and Act / Assert.

The same thing applies to the tests below 👇

Reply:
I agree, that AAA brings more clarity to tests. Sometimes tests are so complex that it's not easy to understand the border between setup and an act 🙂 
I added empty lines to some tests from PR. But some of the tests I find too trivial and I'm not sure if highlighting of their AAA structure will bring some benefit. What do you think about AAA structure highlighting for trivial tests?

Reply:
In such cases, I personally prefer to add it too, for consistency.